controllers/admin/AdminImagesController.php

```diff
@@ -93,7 +93,19 @@ public function __construct()
                         'show' => true,
                         'required' => true,
                         'type' => 'radio',
-                        'choices' => ['jpg' => $this->trans('Use JPEG.', [], 'Admin.Design.Feature'), 'png' => $this->trans('Use PNG only if the base image is in PNG format.', [], 'Admin.Design.Feature'), 'png_all' => $this->trans('Use PNG for all images.', [], 'Admin.Design.Feature')],
+                        'choices' => [
+                            'jpg' => $this->trans('Use JPEG.', [], 'Admin.Design.Feature'),
+                            'png' => $this->trans('Use PNG only if the base image is in PNG format.', [], 'Admin.Design.Feature'),
+                            'png_all' => $this->trans('Use PNG for all images.', [], 'Admin.Design.Feature'),
+                            'webp' => $this->trans('Use WebP for all images.', [], 'Admin.Design.Feature'),
+                            'webp_fb' => $this->trans('Use WebP with fallback to PNG and JPG.', [], 'Admin.Design.Feature'),
+                        ],
+                        'hint' => $this->trans('Use this format to save images internally.', [], 'Admin.Design.Help'),
+                        'desc' => implode('<br>', [
+                            $this->trans('WebP only: [check browser compatibility](https://caniuse.com/webp).', [], 'Admin.Design.Help'),
```

Comment:
Can't we put a real link instead of the markdown syntax here?

Reply:
Is there a reason why you think the link should be visible? :) That might be a best practice I don't know about.

Reply:
I think a real link would be better.

Reply:
Oh, ok, too bad!